### PR TITLE
Adds slug field abstract mixin to Resource and ResourceLink models

### PR DESCRIPTION
**Pull request by Pal0r**
Just saw there was another PR solution for this. [Issue #9](https://github.com/mochaoss/inspace/issues/9). I'll add the related detail views shortly.

### CODE REVIEW

File: inspace/core/views.py [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
 
 
+class ResourceDetailView(DetailView):
+    model = Resource
+    context_object_name = 'resource'
+
+
+class ResourceLinkDetailView(DetailView):
+    model = ResourceLink
+    context_object_name = 'resource'
```
Comment (cacarrara) [inline inspace/core/views.py]: I think here the name should be `resource_link`

Comment (Pal0r) [reply inspace/core/views.py]: You're right. I used that name so we didnt have to refactor that template partial. Let me correct that.

**Comment by Pal0r** [comment]
Adding tests to that save method now.

**Comment by Pal0r** [comment]
Just updates those views and tests. Should be up to 100% coverage now.

**Comment by cacarrara** [comment]
Nice, dude. Thank you for the contribution. Feel welcome to change other things and count on us if you need any help.

**Pull request merged**